README.md

```diff
@@ -227,7 +227,7 @@ This tag identifies the architecture the JDK has been built on and it intended t
 ---
 
 - `variant:`
-Example values: [`hotspot`, `openj9`, `corretto`, `dragonwell`]
+Example values: ['hotpost, `temurin`, `openj9`, `corretto`, `dragonwell`]
```

Comment:
typo `hotspot`?

Question: hotspot variant is still available and what does it mean (openjdk)?

Reply:
It means plain openjdk without any "Temurin" "Eclipse" or "Adoptium" markings on it, or require the `_adopt` git tags when building a tagged version.

This will mean that people can use our scripts in a default way to build their own JDK more easily from a branch of openjdk. The current process trips people up when they're trying to build GA versions particularly becuse of the missing `_adopt` tags and also we don't really want people building their own versions that call themselves "Temurin" by default. Hope that clarifies things.